.snippets/code/tutorials/smart-contracts/launch-your-first-project/test-and-deploy-hardhat/hardhat.config.js

```diff
@@ -0,0 +1,45 @@
+require('@nomicfoundation/hardhat-toolbox');
+
+require('hardhat-resolc');
+require('hardhat-revive-node');
+
+require('dotenv').config();
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+  solidity: '0.8.28',
+  resolc: {
+    version: '1.5.2',
+    compilerSource: 'remix',
```

Comment:
Remix backend has been sunset, so the only option now is to use the local compiler